### ISSUE TITLE
SkeletonBuilder error in 2.2b6 but not in 2.2b5

### ISSUE THREAD

**Issue by KonradDanielewski** [DeepLabCut/DeepLabCut#756]
**Describe the bug**

After extracting frames using deeplabcut.extract_frames(cfp, crop=True) and labeling them using deeplabcut.SkeletonBuilder gives - ValueError: 'vertices' must be a 2D list or array with shape Nx2

**Desktop (please complete the following information about your system):**
 - OS: Windows 10 Pro
 - DeepLabCut Version 2.2b6 both CPU and GPU envs, single animal project

**To Reproduce**
Steps to reproduce the behavior, i.e.:
1. Extract frames with crop set to True
2. Label frames
3. Use deeplabcut.SkeletonBuilder(cfp) or use GUI 'Build skeleton' button
4. See error:

<details><summary>TRACEBACK</summary><p>

```python

In [5]: deeplabcut.SkeletonBuilder(cfp)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-5-41b4df9a73d8> in <module>
----> 1 deeplabcut.SkeletonBuilder(cfp)

~\anaconda3\envs\dlc-cpu\lib\site-packages\deeplabcut\utils\skeleton.py in __init__(self, config_path)
     76                 self.segs.add(tuple(map(tuple, self.xy[pair_sorted, :])))
     77         self.lines = LineCollection(
---> 78             self.segs, colors=mcolors.to_rgba(self.cfg["skeleton_color"])
     79         )
     80         self.lines.set_picker(True)

~\anaconda3\envs\dlc-cpu\lib\site-packages\matplotlib\collections.py in __init__(self, segments, linewidths, colors, antialiaseds, linestyles, offsets, transOffset, norm, cmap, pickradius, zorder, facecolors, **kwargs)
   1224             **kwargs)
   1225
-> 1226         self.set_segments(segments)
   1227
   1228     def set_segments(self, segments):

~\anaconda3\envs\dlc-cpu\lib\site-packages\matplotlib\collections.py in set_segments(self, segments)
   1239             _segments = self._add_offsets(_segments)
   1240
-> 1241         self._paths = [mpath.Path(_seg) for _seg in _segments]
   1242         self.stale = True
   1243

~\anaconda3\envs\dlc-cpu\lib\site-packages\matplotlib\collections.py in <listcomp>(.0)
   1239             _segments = self._add_offsets(_segments)
   1240
-> 1241         self._paths = [mpath.Path(_seg) for _seg in _segments]
   1242         self.stale = True
   1243

~\anaconda3\envs\dlc-cpu\lib\site-packages\matplotlib\path.py in __init__(self, vertices, codes, _interpolation_steps, closed, readonly)
    128         if vertices.ndim != 2 or vertices.shape[1] != 2:
    129             raise ValueError(
--> 130                 "'vertices' must be a 2D list or array with shape Nx2")
    131
    132         if codes is not None:

ValueError: 'vertices' must be a 2D list or array with shape Nx2 


```
</p></details>

**Expected behavior**
Skeleton builder GUI opens

**Additional context**
I tried on two computers, both on CPU and GPU envs with same result. When frames are extracted without cropping everything works. After downgrading to 2.2b5 SkeletonBuilder works.

**Comment by MMathisLab** [comment]
@KonradDanielewski can you confirm which version of pandas and matplotlib you have installed, thanks!

**Comment by KonradDanielewski** [comment]
@MMathisLab I have matplotlib==3.0.3 and pandas==1.0.3 on all 4 environments tested. 

EDIT: Checked also with pandas==1.0.5 and the issue persists.

**Comment by jeylau** [comment]
Hello @KonradDanielewski, I figured what went wrong: the 'single individual' column could accidentally be picked, resulting in undefined segments. This will be fixed with #763.

**Comment by KonradDanielewski** [comment]
Hi, @jeylau . I quickly tested the branch with fix, skeleton builder works fine now. Thanks!
